package.json

```diff
@@ -31,25 +31,7 @@
     "eslint-plugin-sort-class-members": "^1",
     "eslint": "^6",
     "karma-sauce-launcher": "^2",
-    "semantic-release": "^17.1.1",
     "sinon": "^9.0.3"
   },
-  "dependencies": {},
```

Comment:
We don't need this right?  I'm thinking this repo was migrated before the action handled the config for us.

Reply:
Good catch.